debugger/src/debug_adapter.rs

```diff
@@ -4,6 +4,7 @@ use crate::debugger::CoreData;
 use crate::DebuggerError;
 use anyhow::{anyhow, Result};
 use dap_types::*;
+use log::error;
```

Comment:
We use log::error!() directly in code to be more concise about what the macro does. Same goes for log::info!().

Please change the ones you introduced in this PR :)

Reply:
Consider it done :)